src/main/java/io/fabric8/maven/docker/util/DockerFileUtil.java

```diff
@@ -119,6 +119,26 @@ public static FixedStringSearchInterpolator createInterpolator(MojoParameters pa
                 .withExpressionMarkers(delimiters[0], delimiters[1]);
     }
 
+    /**
+     * Fetch the location of docker file in case of simgle configuration
+     *
+     * @return directory containing the Dockerfile
+     */
+    public static String getDockerFileDirectory() {
+        if(checkFileExists("./Dockerfile")) {
+            return new String(".");
```

Comment:
same here: Best to always work with absolute directories, that will work everywhere. With relative directories this is always fragile and it really depends where this method is used.